src/App.js

```diff
@@ -1,64 +1,69 @@
-import React from 'react';
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
 import './App.css';
-import {useState} from 'react';
-import FilterList from './components/FilterList';
+import FilterList from "./components/FilterList";
 
 function App() {
 
-  const [tasks,setTasks] = useState(null);
-  const [status,setStatus] = useState('Pending');
-  const [filter,setFilter] = useState({
-    pending: true,
-    inProgress: true,
-    done: true
-  });
-  const [taskList,setTaskList] = useState([]);
+    const dispatch = useDispatch();
+    const [newTask,setNewTask] = useState({
+        id: '',
+        title: '',
+        status: 'Pending'
+    });
+    const [filter,setFilter] = useState({
+        pending: true,
+        inProgress: true,
+        done: true
+    })
 
 
-  const onChangeHandler = (e) => {
-    setTasks(e.target.value);  
-  }
+    const addTask = (e) => {
+        e.preventDefault(); 
+        dispatch({
+        type: 'ADD_TO_LIST',
+        payload: {
+            id: Date.now(),
+            title: newTask.title,
+            status: newTask.status,
+        }
+    });
+    setNewTask({...newTask, title: '' })
```

Comment:
does this also clear status ? I don't think

Reply:
It is used to set the input filed empty after user has added a task